README.md

```diff
@@ -6,6 +6,62 @@ channel branches (e.g. `channels/8/edge`).
 Read more about the repository structure and build automation in [here](<https://github.com/ubuntu-rocks/.github/blob/main/profile/README.md#-joining-the-ubuntu-rocks-project>).
 
 
+## Image characteristics
+
+This section highlights differences with `eclipse-temurin:8u352-b08-jre-jammy` image (as at the time of writing 8u362 is not yet deployed to Jammy).
+
+### Image size
+
+|Image|Tag|Uncompressed Size| Compressed Size|
+|-----|---|----| ----------------------------|
+| eclipse-temurin|8u352-b08-jre-jammy|221MB|80M|
+| ubuntu/chiselled-jre|8_edge| 123MB|46M |
```

Comment:
```suggestion
| ubuntu/chiselled-jre|8-22.04_edge| 123MB|46M |
```

better add the `22.04` to the tag, as if "tomorrow" we also build jre8 for 24.04, the 8_edge becomes ambiguous and can point to either one